recipes/poethepoet/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "poethepoet" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/poethepoet-{{ version }}.tar.gz
+  sha256: 70b97cb194b978dc464c70793e85e6f746cddf82b84a38bfb135946ad71ae19c
+
+build:
+  entry_points:
+    - poe = poethepoet:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry >=0.12
+    - poetry-core
+    - pip
+  run:
+    - python >={{ python_min }}
+    - pastel >=0.2.0,<0.3.0
+    - tomlkit >=0.6.0,<1.0.0
+
+test:
+  imports:
+    - poethepoet
+  commands:
+    - pip check
+    - poe --help
```

Comment:
Looks like the entry point is failing on Windows. Not sure if it supposed to... Still, it is OK to merge this as noarch.